setup.cfg

```diff
@@ -1,2 +1,6 @@
 [metadata]
 description-file = share/README.md
+
+[pycodestyle]
+ignore = W601
+max-line-length = 160
```

Comment:
max line length we agreed on is 120. There should be code style guide somewhere(it is for erlang, but i guess this point is valid for any of our code). Ofcourse it is discussible.

Reply:
Updated to 120 and added config to allow linting of alertlogic-cli